includes/session_details.js

```diff
@@ -0,0 +1,607 @@
+// Referer URLs in events include URI-formatted codes for some characters e.g. '%20' for ' '. This JS function parses them.
```

Comment:
@Arjun-Kharbanda please can this go in ```data_functions.js``` so that if individual pipelines want to import it and use it themselves then they can?

Reply:
And actually check - it might already be there!

Reply:
Included